From 9357cc3e69e41ad9c56d6419206f0b36603e8191 Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Wed, 20 Nov 2013 22:52:58 +0100
Subject: [PATCH 69/69] r92su: cfg80211_rx_mgmt api change

The api for cfg80211_rx_mgmt has changed with:

  commit 19504cf5f35fbe85db811fce9f4392a0cbdada2f
  Author: Vladimir Kondratiev <qca_vkondrat@qca.qualcomm.com>
  Date:   Thu Aug 15 14:51:28 2013 +0300

    cfg80211: add flags to cfg80211_rx_mgmt()

Since cfg80211_rx_mgmt isn't working anyway, let's just disable
the call.

Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
---
 r92su/rx.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/wireless/r92su/rx.c
+++ b/drivers/net/wireless/r92su/rx.c
@@ -29,6 +29,7 @@
  *
  *****************************************************************************/
 #include <linux/kernel.h>
+#include <linux/version.h>
 #include <linux/etherdevice.h>
 
 #include <net/ieee80211_radiotap.h>
@@ -296,9 +297,11 @@ r92su_rx_handle_mgmt(struct r92su *r92su
 	struct ieee80211_hdr *i3e = (void *)skb->data;
 
 	if (ieee80211_is_mgmt(i3e->frame_control)) {
+#if 0
 		cfg80211_rx_mgmt(&r92su->wdev,
 				 r92su->current_channel->center_freq, 0,
 				 skb->data, skb->len, 0, GFP_ATOMIC);
+#endif
 		dev_kfree_skb_any(skb);
 		return RX_QUEUE;
 	}
